### DISCUSSION TITLE
Synchronisation between accounts on 2 different hosts

### DISCUSSION THREAD

**Discussion by jordivn** [dani-garcia/vaultwarden#5154] [Ideas]
It would be nice if you can synch an specific collection with an account hosted on an different  server.

For instance. I run an server which hold some logins for customers. When an password is updated I inform the customer that the new password is online.
If the customer would like to host his own instance, I would have to get and hold an account on his server as wel. When we can synch, I only have to change the login on one end.

**Comment by BlackDex** [comment]
That is not possible.
The organization keys need to be exactly the same for one.
